docs/content/using/csi_ug_using_sample.md

```diff
@@ -2,38 +2,24 @@
 
 You can use the CSI (Container Storage Interface) driver for running stateful containers with a storage volume provisioned from IBM® block storage systems.
 
-These examples illustrate a basic configuration required for running a stateful container with volumes provisioned on an IBM Spectrum® Virtualize Family storage system.
+These instructions illustrate the general flow for a basic configuration required for running a stateful container with volumes provisioned on storage system.
```

Comment:
`on storage system` ->
`on a storage system`

Reply:
updated in PR 415

Reply:
I [see](https://github.com/IBM/ibm-block-csi-driver/blob/17d919e94d523754830ebae450b82734ea9bfcec/docs/content/using/sample_stateful_container.md) - thanks!